olive/model/handler/snpe.py

```diff
@@ -54,11 +61,11 @@ def prepare_session(
         if device == Device.NPU:
             device = SNPEDevice.DSP
         session_options.device = device
-        return SNPEInferenceSession(self.model_path, self.io_config, session_options)
+        return SNPEInferenceSession(self.model_path, self._io_config, session_options)
```

Comment:
it is supposed to self.io_config both here and at line 68. Since it only expects the four keys.

Reply:
This also works since it the `exclude_none=True` is used but I think `self.io_config` is more correct to use here.

Reply:
good catch

Reply:
updated.